### PR TITLE
Add package recipes for r-mlr3verse and subpackages

### PR DESCRIPTION
**Pull request by pascal-git**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This is my first pull request to conda-forge so I am not 100% sure about all things in the checklist above. The packages I added are all part of the [mlr3verse](https://mlr3.mlr-org.com/) for machine learning in R and are partly interdependent. Therefore I added them in one and the same pull request in the hope that the build will run smoothly this way.

I used the tool in https://github.com/bgruening/conda_r_skeleton_helper to generate the recipes and have build them all locally on a linux machine successfully. The packages are also available from my channel on https://anaconda.org/tux2000/repo

### CODE REVIEW

**Comment by pascal-git** [comment]
With the current recipes the packages are dependent on R 3.5. When Running an example script from https://mlr3book.mlr-org.com/quickstart.html I run in an error at line 3 `learner = lrn("classif.rpart")`  => `could not find function "warningCondition"`. This might be solved by a later version of R so I am currently rebuilding all packages with a newer R version to see if a test script can run then.

**Comment by pascal-git** [comment]
I have built the packages now with R 4.0 and the error mentioned above (https://github.com/conda-forge/staged-recipes/pull/13748#issuecomment-765493518) is fixed. But I don't know how to deal with the linting error...

Ping: @conda-forge/help-r

**Comment by pascal-git** [comment]
@conda-forge-admin, please ping team

File: recipes/r-bbotk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=3.6
```
Comment (dbast) [inline recipes/r-bbotk/meta.yaml:33]: conda-forge anyway currently only builds for R 3.6 and 4.0. Therefore the version constrain can be removed and the linter will be happy.... This applies to all the recipes here... if a R version is not supported by a package, that can be handled via a `skip: True  # ["r_base" == "X.Y"]` entry in the build section. 
```suggestion
    - r-base
```

Comment (isuruf) [reply recipes/r-bbotk/meta.yaml:33]: Yes, revert 31b05e006250c9c4ced51781914b60985c62e3fc

Comment (pascal-git) [reply recipes/r-bbotk/meta.yaml]: I did that now, thanks that you had a look! Now the linter is happy again.

**Comment by dbast** [comment]
12 recipes... all look good... that requires some synchronisation and retriggering CI in the right order so that all get built and uploaded.

**Comment by dbast** [comment]
all packages are now built and uploaded after retriggering CI in the right order... will take another 20 minutes until they are all available.

**Comment by pascal-git** [comment]
Cool. Thanks for the help with this merge request. There are some open automatic pull requests in the individual feedstock repositories. Were they triggered due to problems with dependencies or should I act on them in some way?

**Comment by dbast** [comment]
They are automatic package version updates by the bot... some also failed due to not yet available dependencies... they all need a retrigger ... done that also partially

**Comment by pascal-git** [comment]
Ah, cool. I'll try to check them during the day.

**Pull request merged**